src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -477,14 +477,7 @@ public V getRaw(ResourceLocation key)
         return ret;
     }
 
-    /**
-     * Adds an alias that maps from the name specified by <code>src</code> to the name specified by <code>dst</code>.<p>
-     * Any registry lookups that target the first name will resolve as the second name, if the first name is not present.
-     * @param src The source registry name to alias from.
-     * @param dst The target registry name to alias to.
-     *
-     * TODO: Add as public API in IForgeRegistry and DeferredRegister.
```

Comment:
Only part of this TODO is being addressed by this PR. Either both parts should be or the part about exposing it to DeferredRegister should be kept as a TODO somewhere.

Reply:
The other part is handled (iirc) by @SizableShrimp 's registry refactor, so the todo is mostly moot.  It can't be added to DR without a paradigm shift anyway since it requires moving to a builder-esque pattern.